essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-use-case-software-engineer.md

```diff
@@ -1,2 +1,87 @@
 # \[Draft] Use Case: Software Engineer
 
+
+<!-- Introduction -->
+## Introduction
+Welcome, fellow software engineers! Today, I want to share with you how I use Appflowy to stay organized and productive in my work. As software engineers, we all know how important it is to keep track of our projects, deadlines, and tasks. That's where Appflowy comes in! With its simple and intuitive design, Appflowy makes it easy to stay on top of everything.
+
+![image](https://user-images.githubusercontent.com/89600478/235852123-0105d13e-b05e-4df0-b136-4825ca817351.png)
+
+
+
+<!-- How to Contribute -->
+## Installation
+These are the installation instructions for an end user. If you wish to contribute to AppFlowy or if you just want to view and run directly from source code please visit the [Contribute to AppFlowy section](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy).<br>
+You can view the [System Requirements](https://appflowy.gitbook.io/docs/essential-documentation/install-appflowy/requirements).<br>
+You can go directly to the [Installation methods](https://appflowy.gitbook.io/docs/essential-documentation/install-appflowy/installation-methods).
+
+## Setting Up
+To get started with Appflowy, Once it is installed you'll need to create a workspace. In Appflowy, workspaces are used to group related lists together. For example, you might have a workspace for all your software engineering projects. Within that workspace, you can create categories such as Project Planning, Development Tasks, Bugs and Issues, and Project Completion. This way, you can keep all your work in one place and easily find what you need.
```

Comment:
Could you provide some screenshots of this part?

Reply:
By workspace here I mean dedicated space specifically for your work. Within this workspace, you can create lists, categorize your projects and tasks.....
Let me know if I should change the terminology.

Reply:
Changes has been made, added screenshot plus removed the term workspace instead used pages.